spec/lib/droplet_kit/paginated_resource_spec.rb

```diff
@@ -82,4 +82,17 @@
       end
     end
   end
+
+  describe '#last?' do
+    let(:instance) { DropletKit::PaginatedResource.new(action, resource) }
+
+    it 'returns false when no request is made' do
+      expect(instance.last?).to eq(true)
+    end
+
+    it 'returns true on the last page of results' do
+      instance.each do end
```

Comment:
This empty loop is required to force the resource to iterate over all results in order to get to the last page. If there's a better way to do this, I'd be happy to change this.

Reply:
I think this is totally fine and actually a nice test.